crate_universe/src/context/crate_context.rs

```diff
@@ -284,7 +284,7 @@ impl CrateContext {
             crate_features: annotation.node.features.iter().cloned().collect(),
             deps,
             deps_dev,
-            edition: package.edition.clone(),
+            edition: package.edition.as_str().to_string(),
```

Comment:
This accounts for the thread of changes started in https://github.com/oli-obk/cargo_metadata/pull/183